src/main/scala/is/hail/expr/types/TInterval.scala

```diff
@@ -47,4 +49,13 @@ case class TInterval(pointType: Type, override val required: Boolean = false) ex
   }
 
   override def subst() = TInterval(pointType.subst())
+
+  def loadStart(region: Region, off: Long): Long = representation.loadField(region, off, 0)
+
+  def loadStart(rv: RegionValue): Long = loadStart(rv.region, rv.offset)
```

Comment:
do you need corresponding load statements for includeStart and includeEnd?

Reply:
maybe? I'm actually kind of waffling on whether or not to take these out in favor of Interval.fromRegionValue or something (or maybe something like this already exists) and whether or not to just directly return the boolean for such a load statement. I was going to add them as needed.